Tests/Remora.Discord.Commands.Tests/Responders/InteractionResponderTests.cs

```diff
@@ -214,16 +215,143 @@ public async Task AreExecutedForNotFoundCommands()
                 eventMock.Setup(e => e.User).Returns(new Optional<IUser>(userMock.Object));
                 eventMock.Setup(e => e.Data).Returns(new Optional<IInteractionData>(dataMock.Object));
 
+                var result = await this.Responder.RespondAsync(eventMock.Object);
+                ResultAssert.Unsuccessful(result);  // Should return a CommandNotFoundError
```

Comment:
If that's expected, use an `Assert.IsType` :)

Reply:
Another forgetful comment haha. That could have returned any error that the `CommandService` throws when trying to find the command. This logic has changed with the most recent commits anyway.